apps/sms/test/marionette/lib/views/conversation/view.js

```diff
@@ -6,6 +6,7 @@ var ConversationAccessor = require('./accessors');
 var MessageAccessor = require('./message_accessors');
 var ComposerAccessor = require('../shared/composer_accessors');
 var MenuAccessor = require('../shared/menu_accessors');
+var Tools = require('../shared/tools.js');
```

Comment:
Here I'm not really sure I've done things the way you expected. This certainly allows to factorize the `scriptWith...` stuff, but I feel I should've gone farther. 

On the other hand, if I remove or move the `isMessageInputFocused` and `isRecipientInputFocused`, I would need to declare `messageAccessor` and `composerAccessor` in `Tools` and I didn't feel that worthy. WDYT?